triggers/runner/src/main/scala/com/digitalasset/daml/lf/engine/trigger/Runner.scala

```diff
@@ -46,7 +49,6 @@ import com.daml.script.converter.Converter.{DamlAnyModuleRecord, DamlTuple2, unr
 import com.daml.script.converter.ConverterException
 import com.google.protobuf.empty.Empty
 import com.google.rpc.status.Status
-import com.typesafe.scalalogging.StrictLogging
```

Comment:
Removed usages of this as we will want to control logging via `logback.xml` configuration